tests/utils.go

```diff
@@ -4661,11 +4661,11 @@ func GenerateRandomMac() (net.HardwareAddr, error) {
 	if err != nil {
 		return nil, err
 	}
-	return net.HardwareAddr(append(prefix, suffix...)), nil
+	return append(prefix, suffix...), nil
 }
 
 func getCert(pod *k8sv1.Pod, port string) []byte {
-	randPort := strconv.Itoa(int(4321 + rand.Intn(6000)))
```

Comment:
my golang foo is not strong enough to tell if the casting here is completely meaningless as it seems; a dedicated commit message would reassure me that this is not a fluke.